docs/gallery/interactive/pyfar_filtering.ipynb

```diff
@@ -0,0 +1,286 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "comb_filter.init_state(input_signal.cshape, 'zeros')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, the same result as above can be obtained in blockwise processing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# compute first 128 samples\n",
+    "block_1 = comb_filter.process(input_signal[:, :128])\n",
+    "# compute last 128 samples\n",
+    "block_2 = comb_filter.process(input_signal[:, 128:])\n",
+    "\n",
+    "# plotting the time signals of both blocks\n",
+    "ax = pf.plot.time(block_1, unit='samples', label='block 1')\n",
+    "pf.plot.time(pf.TimeData(block_2.time, np.arange(128)+128), label='block 2')\n",
+    "ax.legend()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Time-variant and block-wise filtering\n",
+    "\n",
+    "If tracking the state of a filter object is enabled as in the example above, a time-variant filter can be realized by changing the filter coefficients between calls of the `process` function. Note that this however, only works if the order of the filter does not change."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# design initial filter coefficients\n",
+    "order = 100\n",
+    "b = np.zeros(order + 1)\n",
+    "b[0] = 1\n",
+    "\n",
+    "# create filter object\n",
+    "comb_filter = pf.FilterIIR(np.stack((b, b), 0), 44100)\n",
+    "comb_filter.init_state(input_signal.cshape, 'zeros')\n",
+    "\n",
+    "# design input signal\n",
+    "block_size = 1024\n",
+    "input_signal = pf.signals.noise(\n",
+    "    order * block_size, sampling_rate=comb_filter.sampling_rate)\n",
+    "\n",
+    "# initialize output signal\n",
+    "output_signal = pf.Signal(\n",
+    "    np.zeros(input_signal.n_samples), input_signal.sampling_rate)\n",
+    "\n",
+    "for nn in range(1, order + 1):\n",
+    "\n",
+    "    # update filter coefficients: increase delay of recursive filter part by\n",
+    "    # one in each block\n",
+    "    comb_filter.coefficients[:, 1, nn] = .95\n",
+    "    if nn > 1:\n",
+    "        comb_filter.coefficients[:, 1, nn-1] = 0\n",
+    "\n",
+    "    # filter current block\n",
+    "    block = comb_filter.process(\n",
+    "        input_signal[:, nn*block_size:(nn+1)*block_size])\n",
+    "\n",
+    "    # write block to output signal\n",
+    "    output_signal.time[:, nn*block_size:(nn+1)*block_size] = block.time\n",
+    "\n",
+    "_ = pf.plot.spectrogram(\n",
+    "    output_signal, window_length=block_size, freq_scale='log')"
```

Comment:
```suggestion
    "pf.plot.spectrogram(\n",
    "    output_signal, window_length=block_size, freq_scale='log');"
```
If you want to suppress the output of the cell you can also use the semi-colon operator